RecoLocalFastTime/Configuration/python/RecoLocalFastTime_EventContent_cff.py

```diff
@@ -19,5 +19,8 @@
     )
 )
 
-from Configuration.Eras.Modifier_phase2_timing_layer_new_cff import phase2_timing_layer_new
-phase2_timing_layer_new.toModify( RecoLocalFastTimeFEVT.outputCommands, func=lambda outputCommands: outputCommands.extend(['drop *_ftlUncalibratedRecHits_*_*','keep *_mtdUncalibratedRecHits_*_*','drop *_ftlRecHits_*_*','keep *_mtdRecHits_*_*']) )
+from Configuration.Eras.Modifier_phase2_timing_layer_tile_cff import phase2_timing_layer_tile
+phase2_timing_layer_tile.toModify( RecoLocalFastTimeFEVT.outputCommands, func=lambda outputCommands: outputCommands.extend(['drop *_ftlUncalibratedRecHits_*_*','keep *_mtdUncalibratedRecHits_*_*','drop *_ftlRecHits_*_*','keep *_mtdRecHits_*_*']) )
+
+from Configuration.Eras.Modifier_phase2_timing_layer_bar_cff import phase2_timing_layer_bar
+phase2_timing_layer_bar.toModify( RecoLocalFastTimeFEVT.outputCommands, func=lambda outputCommands: outputCommands.extend(['drop *_ftlUncalibratedRecHits_*_*','keep *_mtdUncalibratedRecHits_*_*','drop *_ftlRecHits_*_*','keep *_mtdRecHits_*_*']) )
```

Comment:
I was going to make the same comment. It may be even better to keep the old `phase2_timing_layer_new` modifier and just use the new _bar and _tile ones for bar- or -tile-specific changes.

Reply:
I've removed the repetition.